$OpenBSD: patch-plugin_handler_socket_CMakeLists_txt,v 1.1 2019/05/26 22:28:40 jca Exp $

fails to compile plugin handler_socket/auth_pam_tool with -DWITHOUT_SERVER
https://jira.mariadb.org/browse/MDEV-19152

Index: plugin/handler_socket/CMakeLists.txt
--- plugin/handler_socket/CMakeLists.txt.orig
+++ plugin/handler_socket/CMakeLists.txt
@@ -1,5 +1,5 @@
 
-IF(WIN32)
+IF(WIN32 OR WITHOUT_SERVER)
   # Handlersocket does not compile on Windows, compiles but does 
   # not start on FreeBSD.  
   RETURN()
